resources/ory/charts/oathkeeper/templates/cronjob.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ory-oathkeeper-keys-service-account
+  namespace: {{ .Release.Namespace }}
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: {{.Release.Namespace}}
+  name: ory-oathkeeper-keys-job-role
+rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "update", "patch"]
+    resourceNames: ["{{ include "oathkeeper.fullname" . }}-jwks-secret"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["create"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ory-oathkeeper-keys-job-role-binding
+  namespace: {{.Release.Namespace}}
+subjects:
+  - kind: ServiceAccount
+    name: ory-oathkeeper-keys-service-account
+    namespace: {{.Release.Namespace}}
+roleRef:
+  kind: Role
+  name: ory-oathkeeper-keys-job-role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: oathkeeper-jwks-rotator
+spec:
+  schedule: "0 0 * */1 *"
```

Comment:
Default option for a cronjob is `concurrencyPolicy: Allow`. We may want to have it set to `Forbid`